general_configuration/_topics/configuration.md

```diff
@@ -1568,153 +1566,6 @@ To schedule a SmartState Analysis or Compliance Check:
 
 12. Click **Add**.
 
-##### Scheduling a Database Backup
-
-**Note:**
-
-Set `wal_keep_segments` parameter to a value that ensures
-{{ site.data.product.title_short }} scheduled backups finish. Create a test backup
-using a value based on the calculation:
-
-**(\<pg-volume-free-space\>MB/4)/16MB**.
-
-Measure free space in MB and ensure the denominator matches in units.
-
-Setting `wal_keep_segments` to this value will cause the transaction log
-to occupy, at a minimum, one quarter of the current free space. Adjust
-your calculation accordingly upon successful testing.
-
-To set the `wal_keep_segments` value:
-
-1.  SSH to the database appliance
-
-2.  Access the database
-
-        # psql DBNAME USERNAME
-
-3.  Set the `wal_keep_segments` value
-
-        postgres=# ALTER SYSTEM SET wal_keep_segments = <value>;
-
-4.  Reload `wal_keep_segments`
-
-        postgres=# SELECT pg_reload_conf();
-
-On successful backup, revise the value for `wal_keep_segments` to save
-space based on transaction log size.
```

Comment:
Note, the above information on `wal_keep_segments` seems related to database backups and not schedules of these backups.  @dehawkins512 do we have documentation on doing database/postgres backups using pg basebackup or pg_dump?  I wonder if this section should move there.

Reply:
@jrafanie Backing up and Restoring a database is documented here:
https://www.manageiq.org/docs/reference/latest/general_configuration/#backing-up-and-restoring-a-database-for-an-appliance-based-environment
The only docs that mention pg_dump are in the Install on Kubernetes and migrating from appliances topics here:
https://www.manageiq.org/docs/reference/morphy/installing_on_kubernetes/index.html

Reply:
@jrafanie Can we confirm the information on `wal_keep_segments` is used in any DB backup other than a scheduled one?
I don't see where the ManageIQ docs mention using pg basebackup or pg_dump.

Reply:
@dehawkins512 I did some quick googling and I think it's no longer in postgres 13.  I guess we can remove it.  I think a replacment concept of `wal_keep_size` was added.  See: https://www.depesz.com/2020/07/27/waiting-for-postgresql-14-rename-wal_keep_segments-to-wal_keep_size/

See also: https://www.postgresql.org/docs/13/runtime-config-replication.html
(wal_keep_size added, wal_keep_segments removed)
vs:

https://www.postgresql.org/docs/12/runtime-config-replication.html
(still had wal_keep_segments)